src/modules/todolist.js

```diff
@@ -0,0 +1,42 @@
+import Task from './task.js';
+
+class ToDoList {
+  constructor() {
+    this.list = [];
+  }
+
+  addNewTask = (index = null, description = null, isCompleted = null) => {
+    if (description) {
+      const newTaskIndex = index || this.list.length + 1;
+      const newTaskIsCompleted = isCompleted || false;
```

Comment:
I think you have already declared "isCompleted" to false when the class **Task** was created, so you could avoid repeating this declaration.